author.tf

```diff
@@ -280,6 +280,14 @@ module "author" {
         "name": "REACT_APP_FIREBASE_API_KEY",
         "value": "${var.author_firebase_api_key}"
       }
```

Comment:
```suggestion
      },
```

Reply:
Good spot we literally just found these haha